pkg/graphql/request_test.go

```diff
@@ -76,6 +76,26 @@ func TestRequest_Print(t *testing.T) {
 	assert.Equal(t, query, bytesBuf.String())
 }
 
+func TestRequest_IsNormalized(t *testing.T) {
+	request := Request{
```

Comment:
I found `TestRequest_Normalize`, it also checks `isNormalized` field. So we shouldn't need this unit test.

Reply:
Here it is https://github.com/TykTechnologies/graphql-go-tools/blob/92655e8c2e963252c0e5a742925d0bc0d536e763/pkg/graphql/normalization_test.go#L26